src/ui/utils/testRuns.ts

```diff
@@ -75,6 +75,8 @@ export function groupRecordings(tests: TestRunTestWithRecordings[]): TestGroups
         } else {
           addToTestGroup(failedRecordings, test);
         }
+      } else if (test.result === "flaky") {
+        addToTestGroup(flakyRecordings, test);
```

Comment:
I'll open a follow up to look into this. I think we can simplify this check once the backend change lands but I'm not confident that's the case until I can test it with production data. If everything goes as planned, we could reduce this to a `switch` since each branch would only be adding to a single map and not choosing between two.

Reply:
Opened SCS-1617